content/en/docs/components/training/overview.md

```diff
@@ -41,11 +33,68 @@ for each ML framework:
 | ML Framework | Custom Resource                                      |
 | ------------ | ---------------------------------------------------- |
 | PyTorch      | [PyTorchJob](/docs/components/training/pytorch/)     |
-| Tensorflow   | [TFJob](/docs/components/training/tftraining/)       |
+| TensorFlow   | [TFJob](/docs/components/training/tftraining/)       |
 | XGBoost      | [XGBoostJob](/docs/components/training/xgboost/)     |
 | MPI          | [MPIJob](/docs/components/training/mpi/)             |
 | PaddlePaddle | [PaddleJob](/docs/components/training/paddlepaddle/) |
 
+## Architecture
+
+This diagram shows the major features of Training Operator and supported ML frameworks.
+
+<img src="/docs/components/training/images/training-operator-overview.drawio.png"
+  alt="Training Operator Overview"
+  class="mt-3 mb-3">
+
+Training Operator is responsible to schedule the appropriate Kubernetes workloads to implement
+various distributed training strategies for different ML frameworks. The following examples show
+how Training Operator allows to run distributed PyTorch and TensorFlow on Kubernetes.
+
+### Distributed Training for PyTorch
+
+This diagram shows how Training Operator creates PyTorch workers for
+[ring all-reduce algorithm](https://tech.preferred.jp/en/blog/technologies-behind-distributed-deep-learning-allreduce/).
```

Comment:
I'm OK here to put the allreduce algorithm as an example to show distributed training with PyTorch, however, I've a little bit concerns for two aspects,

* if we limit the scope to ring allreduce, horovod would be the best practice
* different and more complicate collective algorithms can be used with PyTorch by adopting NCCL or alternatives, all can be done with PyTorchJob configuration, e.g. [accelerate](https://github.com/huggingface/accelerate)

My personal suggestion would be to emphasize this as example and the operator is open to all popular framework and user cases.

Reply:
It's good points. I think that we can mention in the top of `##Architecture` section.

Reply:
I agree that Horovod will be preferable implementation for ring all-reduce, but PyTorch DDP docs also can do it as it says here: https://pytorch.org/tutorials/beginner/ddp_series_theory.html#what-is-distributed-data-parallel-ddp.

>My personal suggestion would be to emphasize this as example and the operator is open to all popular framework and user cases.

Yes, I tried to emphasize it here: https://github.com/kubeflow/website/pull/3678/files#diff-768cd67df75edc45450aa16fb45d81da4b5fc0dfdcd2457c2bd81cc16a10e3e7R72-R73
@kuizhiqing Do you have any other suggestion how we can explain it ?